templates/monitoring/grafana/05-grafana.yaml

```
@@ -152,4 +163,3 @@ spec:
         port:
           number: 3000
         host: grafana
```

not sure why this is not showing in my editor
```suggestion
        host: grafana

```